app/models/post.rb

```diff
@@ -1074,7 +1076,7 @@ def link_post_uploads(fragments: nil)
 
     UploadReference.transaction do
       UploadReference.where(target: self).delete_all
-      UploadReference.insert_all(upload_references) if upload_references.size > 0
+      result = UploadReference.insert_all(upload_references) if upload_references.size > 0
```

Comment:
We're not using `result` further here, are we?

Reply:
Nope I must of added it while puts debugging.